RecoEcal/Configuration/python/RecoEcal_EventContent_cff.py

```diff
@@ -107,11 +41,54 @@
-        e.toModify( ec, func=lambda outputCommands: outputCommands.extend(['keep recoSuperClusters_correctedIslandBarrelSuperClusters_*_*',
-                                                                           'keep recoSuperClusters_correctedIslandEndcapSuperClusters_*_*'
-                                                                           ])
-                    )
-    for ec in [RecoEcalRECO.outputCommands, RecoEcalFEVT.outputCommands]:
-        e.toModify( ec, func=lambda outputCommands: outputCommands.extend(['keep recoCaloClusters_islandBasicClusters_*_*'])
-                    )
+    e.toModify( RecoEcalAOD.outputCommands, 
+                func=lambda outputCommands: outputCommands.extend(['keep recoSuperClusters_correctedIslandBarrelSuperClusters_*_*',
+                                                                   'keep recoSuperClusters_correctedIslandEndcapSuperClusters_*_*'])
+              )
+# RECO content
+RecoEcalRECO = cms.PSet(
+    outputCommands = cms.untracked.vstring(
+	# Barrel clusters
+        'keep *_hybridSuperClusters_*_*',
+        'keep recoSuperClusters_correctedHybridSuperClusters_*_*',
+	# Endcap clusters
+        'keep *_multi5x5SuperClusters_*_*',
+        'keep recoSuperClusters_multi5x5SuperClusters_*_*',
+        'keep recoSuperClusters_multi5x5SuperClustersWithPreshower_*_*',
+        # Particle Flow superclusters
+        'keep *_particleFlowSuperClusterECAL_*_*',
+        'keep *_particleFlowSuperClusterOOTECAL_*_*',
+	# DROP statements # not used
+        #'drop recoClusterShapes_*_*_*', 
+        #'drop recoBasicClustersToOnerecoClusterShapesAssociation_*_*_*',
+        #'drop recoBasicClusters_multi5x5BasicClusters_multi5x5BarrelBasicClusters_*',
+        #'drop recoSuperClusters_multi5x5SuperClusters_multi5x5BarrelSuperClusters_*')
+        )
+)
+RecoEcalRECO.outputCommands.extend(RecoEcalAOD.outputCommands)
+_phase2_hgcal_scCommands = ['keep *_particleFlowSuperClusterHGCal_*_*',
+                            'keep *_particleFlowSuperClusterHGCalFromMultiCl_*_*']
+phase2_hgcal.toModify(RecoEcalRECO,
+    outputCommands = RecoEcalRECO.outputCommands + _phase2_hgcal_scCommands)
+
+for e in [pA_2016, peripheralPbPb, pp_on_AA_2018, pp_on_XeXe_2017, ppRef_2017]:
+    e.toModify( RecoEcalRECO.outputCommands,
+                func=lambda outputCommands: outputCommands.extend(['keep recoCaloClusters_islandBasicClusters_*_*'])
+              )
+# Full Event content 
+RecoEcalFEVT = cms.PSet(
+    outputCommands = cms.untracked.vstring(
+	# Hits
+	'keep *_reducedEcalRecHitsEB_*_*',
+        'keep *_reducedEcalRecHitsEE_*_*',
+        'keep *_reducedEcalRecHitsES_*_*', 
+        'keep *_interestingEcalDetId*_*_*', 
+        'keep *_ecalWeightUncalibRecHit_*_*', 
+        'keep *_ecalPreshowerRecHit_*_*', 
+	# Barrel clusters
+        'keep *_correctedHybridSuperClusters_*_*',
+	# Endcap clusters
+        'keep *_multi5x5*_*_*',
```

Comment:
Can this get expanded without the wild card (and without blowing the list too much)?

Reply:
If this is in FEVT, I think that an exception can be made, considering that this datatier is more debug-like than something for sustained production

Reply:
I've found 4 modules in RECO/FEVT event contents : 
**multi5x5BasicClustersCleaned 
multi5x5SuperClusters 
multi5x5SuperClustersWithPreshower
multi5x5PreshowerClusterShape** 
with different types and labels.
```
vector<reco::CaloCluster> multi5x5BasicClustersCleaned * *
vector<reco::SuperCluster> multi5x5BasicClustersCleaned * *
vector<reco::CaloCluster> multi5x5SuperClusters * *
vector<reco::SuperCluster> multi5x5SuperClusters * *
vector<reco::PreshowerCluster> multi5x5SuperClustersWithPreshower * *
vector<reco::SuperCluster>  multi5x5SuperClustersWithPreshower * *
vector<reco::PreshowerClusterShape> multi5x5PreshowerClusterShape * *
```